src/lightkurve/io/detect.py

```diff
@@ -47,6 +48,10 @@ def detect_filetype(hdulist: HDUList) -> str:
     if "mit/qlp" in hdulist[0].header.get("origin", "").lower():
         return "QLP"
 
+    # Is it a GSFC-ELEANOR-LITE light curve?
+    if (hdulist[0].header.get("AUTHOR") == 'Adina D. Feinstein') and (hdulist[0].header.get("LITE") == True):
```

Comment:
Ah I see that you used both, maybe both is ok. I am worried that `hdulist[0].header.get("LITE")` will return something weird and this will break if the keyword doesn't exist?

@vbkostov can you show that if you run this on a fits file that doesn't have that keyword in the header that this statement works?